docs/xks/operator-guide/operator-guide.md

```diff
@@ -68,12 +105,153 @@ Lets create the first terraform state, in this case governance.
 
 `make prepare ENV=dev DIR=governance`
 
-### CI/CD
+You will need to run the prepare command for each separate terraform folder.
+
+### Configure governance
+
+After defining the variables and you have have applied your config
+
+```shell
+make plan ENV=dev DIR=governance
+# If everything looks good
+make apply ENV=dev DIR=governance
+```
+
+### Configure core
+
+Get a CIDR network for your AKS env per env.
+Define in core/variables/env.tfvars
+
+### Configure AKS cluster
+
+- How big should your cluster be?
+- Which version of kubernetes should you run?
+- What DNS zone should you use?
+- What SKU tier should your cluster have?
+- What size should your k8s nodes have?
 
-We will have one CI/CD pipeline per DIR.
+All of this is configured under aks/variables/prod.tfvars
+
+```prod.tfvars
+environment = "dev"
+dns_zone    = "dev.aks.xenit.io"
+
+aks_config = {
+  kubernetes_version = "1.20.7"
+  sku_tier           = "Free"
+  default_node_pool = {
+    orchestrator_version = "1.20.7"
+    vm_size              = "Standard_D2as_v4"
+    min_count            = 1
+    max_count            = 1
+    node_labels          = {}
+  },
+  additional_node_pools = [
+    {
+      name                 = "standard2"
+      orchestrator_version = "1.20.7"
+      vm_size              = "Standard_D2as_v4"
+      min_count            = 1
+      max_count            = 4
+      node_labels          = {}
+      node_taints          = []
+    },
+  ]
+}
+```
+
+> Notice the vm_size = Standard_D2as_v4
+
+### GitOps using flux
+
+If you want flux to manage your GitOps repo from the get go you can enable this in aks/variables/common.tfvars
+In my case I will have flux manage a namespace called monitor and sync a repo under monitor-gitops.
+
+You need to create the repository in Azure Devops that you link to before applying this terraform.
+The repository can be empty.
+
+You will also need to create a separate repository for `fleet-infra`, this repo is used to store flux config.
+> This repo cannot be empty and needs a README file or something similar to work as intended before you run terraform.
+
+```common.tfvars
+namespaces = [
+  {
+    name                    = "monitor"
+    delegate_resource_group = true
+    labels = {
+      "terraform" = "true"
+    }
+    flux = {
+      enabled = true
+      azure_devops = {
+        org  = "organization1"
+        proj = "project1"
+        repo = "monitor-gitops"
+      }
+      github = {
+        repo = ""
+      }
+    }
+  }
+]
+```
+
+## CI/CD
```

Comment:
In this case it's for the operator since this is for terraform and part of the platform so I think it should be under the operator guide.
But I renamed it Terraform CI/CD to make it a bit clearer.